### PR TITLE
[OBSOLETE] Render temporary html in PDF_DIR_PATH + removed encode() for Python3

### PR DESCRIPTION
**Pull request by keklesurvivant**
Hello,
For my needs, I had to implement the suggestion from issue #2 
At the same time, I had to remove the call to encode('utf-8') when using render_template to make it work on my Python 3.5 environment.
Best regards

### CODE REVIEW

File: flask_wkhtmltopdf.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def _maybe_decorate(condition, decorator):
 
 
     @_maybe_decorate(use_celery, celery.task())
-    def render_template_to_pdf(self, template_name_or_list, save=False, download=False, **context):
+    def render_template_to_pdf(self, template_name_or_list, options="", save=False, download=False, **context):
```
Comment (chris-griffin) [inline flask_wkhtmltopdf.py:84]: Can you add documentation for `options` below

**Review by chris-griffin — COMMENTED** [review]
Looks good pending one documentation comment. The tests are failing because [GNA no longer hosts the wkhtmltopdf releases](https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3390). I'll update that.

File: flask_wkhtmltopdf.py (hunk 1 above)
Comment (keklesurvivant) [reply flask_wkhtmltopdf.py:84]: Doc updated. Sorry for the delay. Hope everything is ok.

**Comment by keklesurvivant** [comment]
Please cancel this pull request. I observed a regression on my windows environment, and wrongly reverted the commit. I will submit a proper Pull request once fully tested on my side. Sorry for the inconvenience.

**Pull request closed without merging**